src/msg/async/ProtocolV1.cc

```diff
@@ -1025,8 +1025,12 @@ CtPtr ProtocolV1::handle_message_footer(char *buffer, int r) {
   if (connection->delay_state) {
     double delay_period = 0;
     if (rand() % 10000 < cct->_conf->ms_inject_delay_probability * 10000.0) {
-      delay_period =
-          cct->_conf->ms_inject_delay_max * (double)(rand() % 10000) / 10000.0;
+      if (cct->_conf->ms_inject_fixed_delay > 0.0) {
```

Comment:
looks like we already have `ms_inject_internal_delays`, isn't that enough?

Reply:
It seems that ms_inject_internal_delays works when connecting, not every normal message. Correct me if I misunderstand, thanks.